--- configure~	2014-11-09 14:15:29.000000000 -0800
+++ configure	2016-09-19 18:02:25.047099818 -0700
@@ -4211,7 +4211,7 @@
     VERSION_OPT=''
   fi
 
-  LINK_SHARED="$LD"' -o $$@ -soname libtecla$$(SUFFIX).so.'${MAJOR_VER}' -shared '$VERSION_OPT' $$(LIB_OBJECTS) $$(LIBS) -lc'
+  LINK_SHARED="$LD"' -o $$@ --build-id -soname libtecla$$(SUFFIX).so.'${MAJOR_VER}' -shared '$VERSION_OPT' $$(LIB_OBJECTS) $$(LIBS) -lc'
   SHARED_CFLAGS="-fpic"
 
 
